README.md

```diff
@@ -81,6 +86,20 @@ $ driftctl scan --from tfstate://terraform.tfstate
 $ driftctl scan --from tfstate+s3://my-bucket/path/to/state.tfstate
 ```
 
+Or with ou docker image:
+```bash
+docker run \
+-v ~/.aws:/app/.aws:ro \
+-v $(pwd)/terraform.tfstate:/app/terraform.tfstate:ro \
+-v ~/.driftctl:/app/.driftctl \
+-e AWS_PROFILE=cloudskiff \
+cloudskiff/driftctl scan ## with the same option as the cli version
```

Comment:
`## with the same option as the binary version`

I would replace "cli" with "binary" as it remain a CLI even if launched with docker 😉

Reply:
:+1: